src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -677,8 +683,8 @@ void ModControllableAudio::processSRRAndBitcrushing(StereoSample* buffer, int32_
 				                  + multiply_32x32_rshift32_rounded(currentSample->l, strength2 << 9);
 				grabbedSample.r = multiply_32x32_rshift32_rounded(lastSample.r, strength1 << 9)
 				                  + multiply_32x32_rshift32_rounded(currentSample->r, strength2 << 9);
-				grabbedSample.l &= bitCrushMaskForSRR;
-				grabbedSample.r &= bitCrushMaskForSRR;
+				grabbedSample.l &= bitCrushMask;
```

Comment:
It drops a bit during the multiplication above this line so the difference is to compensate for that. Dunno if it makes sense to do that though?

Reply:
So, I think this makes sense. As you figured out, the SRR adds one bit of reduction. 0 to 1 is hard to hear, but each subsequent step is obvious. This keeps the audible crush the same with and without SRR - and old instruments sounding the same.